packages/stark-ui/src/modules/table/entities/column-properties.intf.ts

```diff
@@ -86,4 +86,16 @@ export interface StarkTableColumnProperties {
 	 * Priority of the column. Default: PERSIST
 	 */
 	priority?: StarkTableColumnPriority;
+
+	/**
+	 * Whether the column should be shown on the right side of the table (stickyEnd)
+	 * Default: false
+	 */
+	stickyEnd?: boolean;
```

Comment:
Please also add a new table demo in the showcase where you use these 2 new props 😉